contrib/segmentio/kafka.go.v0/kafka.go

```diff
@@ -61,8 +62,10 @@ func (r *Reader) startSpan(ctx context.Context, msg *kafka.Message) ddtrace.Span
 		tracer.Tag(ext.Component, componentName),
 		tracer.Tag(ext.SpanKind, ext.SpanKindConsumer),
 		tracer.Tag(ext.MessagingSystem, "kafka"),
+		tracer.Tag(ext.KafkaBootstrapServers, strings.Join(r.Config().Brokers, ",")),
```

Comment:
Can these brokers change during the lifetime of this `Reader`? If not, it might be worth adding a benchmark and calculating this value exactly once since this `startSpan` is in the hotpath for kafka reading

Reply:
Changes were implemented by calculating value only once upon wrapping. Wrote simple benchmark and improved efficiency by ~100 ns/op so good catch!